src/Features/LanguageServer/Protocol/Handler/References/FindImplementationsHandler.cs

```diff
@@ -40,7 +40,7 @@ public async Task<LSP.Location[]> HandleRequestAsync(LSP.TextDocumentPositionPar
 
             var locations = ArrayBuilder<LSP.Location>.GetInstance();
 
-            var findUsagesService = document.GetRequiredLanguageService<IFindUsagesService>();
+            var findUsagesService = document.GetRequiredLanguageService<IFindUsagesLSPService>();
```

Comment:
Like FAR, gotoimpl should be using the lsp service.  See https://github.com/dotnet/roslyn/blob/40186a1f10742ac6e610e2e84787a07fbbb189ed/src/Features/Core/Portable/FindUsages/AbstractFindUsagesService_FindReferences.cs#L44-L49